### DISCUSSION TITLE
Textadept window border

### DISCUSSION THREAD

**Discussion by potty-trained** [orbitalquark/textadept#320] [Q&A] [answered]
I want dark mode.
this is not darkmode
how tf do i make the window border dark so its true dark mode

**Comment by orbitalquark** [comment, marked as answer]
Textadept's window and GUI widgets inherit the system style; it cannot change window borders.

**Reply by potty-trained** [reply]
if I may be so bold to ask, since you have CURSES could you add a way to edit the border, or perhaps make an official plugin

perhaps a tutorial as to how to do so on ones own terms?

**Comment by orbitalquark** [comment]
No, sorry, this is out of scope. I don’t know which OS you’re on, but on Linux the window manager is responsible for drawing window borders. Try changing your WM settings.

Curses refers to the terminal version of Textadept, which has no concept of window orders.

**Reply by potty-trained** [reply]
I am on windows, if that changes "I don’t know which OS you’re on, but on Linux the window manager is responsible for drawing window borders. Try changing your WM settings."

**Comment by orbitalquark** [comment]
I see. You might try looking into how to enable dark mode on Windows then. Note that Textadept does not (yet, I hope) update its color theme to match its host OS dark mode settings, so you'll have to manually set the editor theme to dark.

**Reply by potty-trained** [reply]
i am in dark mode, i have the dark mode theme, just the window border is white and it looks horrible with the rest of the app being dark mode

**Comment by orbitalquark** [comment]
Right, sorry. I don't know how to fix it :(

**Reply by potty-trained** [reply]
hey man dw its a great app regardless, i like it very much and serves its purpose well

**Comment by snoopy** [comment]
On Linux TA seems to already respect dark mode if the environment/WM is set to it. On Windows it apparently has to be set by the application. I don't have a lot of experience with Qt but this might help: https://github.com/Alexhuszagh/BreezeStyleSheets
